src/main/kotlin/org/wycliffeassociates/otter/jvm/resourcecardpage/view/ResourcesView.kt

```diff
@@ -0,0 +1,20 @@
+package org.wycliffeassociates.otter.jvm.resourcecardpage.view
+
+import tornadofx.View
+import tornadofx.Workspace
+import tornadofx.removeFromParent
+import tornadofx.vbox
+
+class ResourcesView : View() {
+
+    override val root = vbox {}
+
+    var activeFragment: Workspace = Workspace()
```

Comment:
AFAIK we can't have multiple workspaces

The resources view should not be docking fragments